ckanext/ontario_theme/templates/internal/group/index.html

```diff
@@ -1,19 +1,23 @@
 {% ckan_extends %}
 
+{% block breadcrumb_content %}
+  <li class="active">{% link_for _('Collections'), controller='group', action='index', named_route=group_type + '_index' %}</li>
```

Comment:
Instead of hardcoding the collection type name we could switch this to something like

`<li class="active">{% link_for _(group_type.title()), controller='group', action='index', named_route=group_type + '_index' %}</li>`

This allows us to make one change and have the terms (i.e. "Collection", "Group", "New thing") updated automatically.

This should apply to most template updates made in this PR and hopefully help prevent changes for future us.

Reply:
It was agreed to make all these changes to ckanext-scheming core instead.